src/components/UI/UFormButton/UFormButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { ButtonHTMLAttributes } from 'react';
+
+export enum ButtonType {
+  SUBMIT = 'submit',
+  BUTTON = 'button',
+}
+
+interface IFormButton extends ButtonHTMLAttributes<HTMLButtonElement> {
+  isDisabled: boolean;
+  type: ButtonType;
+  className?: string;
+  onClick?: () => void;
+  icon?: React.ReactNode;
+  text?: string;
+  customClassName?: string;
+}
+
+const UFormButton: React.FC<IFormButton> = ({
+  type,
+  isDisabled = true,
+  text,
+  onClick,
+  className,
```

Comment:
```suggestion
  className = "",
```
You should use default param because if you don't pass className prop then button class will contain "undefined" class by default